### ISSUE TITLE
Doesn't work in Sublime 3 ?

### ISSUE THREAD

**Issue by energylevels** [jonschlinkert/sublime-monokai-extended#93]
This doesn't to work in Sublime any more - tried with package installer and installing manually

**Comment by cemerson** [comment]
Yes very sad that the # lines in my markdown are no longer colored. :( Hoping to find someone who has this working in ST4
